common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -403,10 +403,17 @@ def test_can_reset_attempts(self):
 
 
 @attr(shard=7)
+@ddt.ddt
 class EntranceExamGradeTest(BaseInstructorDashboardTest):
     """
     Tests for Entrance exam specific student grading tasks.
     """
+    admin_buttons = (
```

Comment:
**Note to reviewers:** Changed these tests to use `ddt` instead - since they were repeated code for the various admin buttons.